src/forms/BoundSelectField.tsx

```diff
@@ -5,12 +5,15 @@ import { HasIdAndName, Optional } from "src/types";
 import { maybeCall } from "src/utils";
 import { defaultLabel } from "src/utils/defaultLabel";
 import { useTestIds } from "src/utils/useTestIds";
+import { Css } from "..";
```

Comment:
I'm a bit worried about this import. I think i've seen this before and it blew up `internal-frontend`. Can we make this an absolute path?

Reply:
Sure but I may end up closing this PR altogether anyway. Got a little too ambitious with this one.

Reply:
Okay no problem. I was going to ask if this was a new pattern introduced somewhere. But now just seeing the messages between you and Stephen.